src/compose/network-manager.ts

```diff
@@ -40,17 +42,45 @@ export class NetworkManager {
 		return this.getAll().filter((network: Network) => network.appId === appId);
 	}
 
-	public get(network: { name: string; appId: number }): Bluebird<Network> {
-		return Network.fromNameAndAppId(
-			{
-				logger: this.logger,
-				docker: this.docker,
-			},
-			network.name,
-			network.appId,
+	public async get(network: {
+		name: string;
+		appId: number;
+	}): Bluebird<Network> {
+		const dockerNet = await this.docker
+			.getNetwork(`${network.appId}_${network.name}`)
```

Comment:
Might be nice to have a `Network.generateDockerName(appId, name)` like you did for Volume

Reply:
+1 here

Though I would rather have it like
```
this.docker.getNetwork(network.dockerName())
```

Reply:
@roman-mazur it's a lot more flexible to have a static function when there's no need to create an object for the method's task.